docs/source/examples/combine_techniques.md

```diff
@@ -0,0 +1,107 @@
+
+```{code-cell} ipython3
+circuit = generate_rb_circuits(2, 5)[0]
+```
+
+## Noise model and executor
+
+The noise in this example is a combination of depolarizing and readout errors, the latter of which are modeled as bit flips immediately prior to measurement. We use an [`Executor`](../guide/executors.md) function to execute the quantum circuit with the noise model applied.
+
+```{code-cell} ipython3
+def execute(circuit: cirq.Circuit, noise_level: float = 0.005, p0: float = 0.05) -> MeasurementResult:
+    """Execute a circuit and return a ``MeasurementResult``, with depolarizing noise of strength
+    ``noise_level`` and readout errors with probability ``p0``."""
+    measurements = circuit[-1]
+    circuit =  circuit[:-1]
+    circuit = circuit.with_noise(cirq.depolarize(noise_level))
+    circuit.append(cirq.bit_flip(p0).on_each(circuit.all_qubits()))
+    circuit.append(measurements)
+
+    simulator = cirq.DensityMatrixSimulator()
+
+    result = simulator.run(circuit, repetitions=10000)
+    bitstrings = np.column_stack(list(result.measurements.values()))
+    return MeasurementResult(bitstrings)
+```
+
+## Observable
+
+We then obtain a unmitigated expectation value of the `ZI` + `ZZ` observable. In the ideal case the result is 2, but as we will see, the unmitigated result is impacted by depolarizing and readout errors.
+
+```{code-cell} ipython3
+obs = Observable(PauliString("ZI"), PauliString("IZ"))
+noisy = raw.execute(circuit, execute, obs)
+```
+
+```{code-cell} ipython3
+from functools import partial
+ideal = raw.execute(circuit, partial(execute, noise_level=0, p0=0), obs)
+f"Error without mitigation: {abs(ideal - noisy) :.5f}"
+```
+
+Next we generate the inverse confusion matrix and apply readout error mitigation (REM). More information on generating the inverse confusion matrix is available in the [REM theory](../guide/rem-5-theory.md) section of the user guide. 
+
+```{code-cell} ipython3
+from mitiq import rem
+p0 = p1 = 0.05
+icm = rem.generate_inverse_confusion_matrix(2, p0, p1)
+rem_executor = rem.mitigate_executor(execute, inverse_confusion_matrix=icm)
+
+rem_result = obs.expectation(circuit, rem_executor)
+f"Error with REM: {abs(ideal - rem_result) :.5f}"
+```
+
+We can see that REM improves the results, but errors remain. For comparison, we then apply ZNE (without REM).
+
+```{code-cell} ipython3
+from mitiq import zne
+
+zne_executor = zne.mitigate_executor(execute, observable=obs)
+zne_result = zne_executor(circuit)
+f"Error with ZNE: {abs(ideal - zne_result) :.5f}"
+```
+
+Finally, we apply a combination of REM and ZNE. REM is applied first to minimize the impact of measurement errors on the extrapolated result in ZNE.
+
+```{code-cell} ipython3
+combined_executor = zne.mitigate_executor(rem_executor, observable=obs)
+
+combined_result = combined_executor(circuit)
+f"Error with REM + ZNE: {abs(ideal - combined_result) :.5f}"
+```
```

Comment:
Wow cool to see this has such a small error compared to unmitigated! I noticed above it says the ideal expectation value is 2, but it looks like we're approaching 0 as the ideal value.

Reply:
I changed the result statements back to expectation values instead of errors for clarity.